README.md

```diff
@@ -1,76 +1,72 @@
-     -->
     <com.matthew.passwordinput.lib.PasswordInputView
-        android:layout_width="match_parent"
-        android:layout_height="50dp"
-        android:layout_marginTop="10dp"
-        android:background="@null"
-        android:padding="1dp"
-        android:text="123"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:pwv_haveInputBorderColor="@color/colorAccent"
-        app:pwv_pwdStyle="plaintext"
-        app:pwv_radius="10dp"
-        app:pwv_spacing="12dp" />
-
-```
-#### 设置监听器
-```java
-    passwordView.setInputListener(new PasswordInputView.InputListener() {
-        @Override
-        public void onInputCompleted(String text) {
-            // 输入完成后回调
-        }
-    });
+         ohos:id="$+id:passwordView"
+         ohos:width="match_parent"
+         ohos:height="50vp"
+         ohos:top_margin="10vp"
+         ohos:background_element="@null"
+         ohos:text_input_type="pattern_number"
+         ohos:padding="1vp"
+         ohos:text="12"
+         ohos:text_size="32fp"
+         app:layout_constraintLeft_toLeftOf="parent"
+         app:layout_constraintRight_toRightOf="parent"
+         app:layout_constraintTop_toTopOf="parent"
+         app:pwv_borderColor="$color:colorPrimary"
+         app:pwv_haveInputBorderColor="$color:colorAccent"
+         app:pwv_pwdColor="$color:colorAccent"
+         app:pwv_pwdStyle="plaintext"
+         app:pwv_radius="10vp"
+         app:pwv_borderStyle="line"
+         app:pwv_maxLength="4"
+         app:pwv_spacing="12vp" />
 ```
-#### 属性参考
-```xml
-    <declare-styleable name="PasswordInputView">
-        <attr name="pwv_maxLength" format="integer" /> // 最大长度
-
-        <attr name="pwv_borderColor" format="color" /> // 边框颜色
-        <attr name="pwv_pwdColor" format="color" /> // 密码颜色
-        <attr name="pwv_haveInputBorderColor" format="color" /> // 已输入部分边框的颜色
 
-        <attr name="pwv_strokeWidth" format="dimension" /> // 边框宽度
-        <attr name="pwv_radius" format="dimension" /> // 圆角半径
-        <attr name="pwv_spacing" format="dimension" /> // 每个密码框之间的间距
-        <attr name="pwv_asterisk" format="string" />
-        // 当密码风格为星号风格时，可以用任意字符替换星号，替换的字符为pwv_asterisk的第一个字符
-
-        <attr name="pwv_borderStyle" format="enum"> // 边框风格 方框 和 下划线
-            <enum name="box" value="0" />
-            <enum name="line" value="1" />
-        </attr>
-        <attr name="pwv_pwdStyle" format="enum"> // 密码风格 圆点、星号、明文
-            <enum name="circle" value="0" />
-            <enum name="asterisk" value="1" />
-            <enum name="plaintext" value="2" />
-        </attr>
-    </declare-styleable>
+#### Set up the listener:
+```java
+      passwordView.setInputListener(new PasswordInputView.InputListener() {
+          @Override
+          public void onInputCompleted(String text) {
+              // Call back after input is complete
+          }
+      });
```

Comment:
Please list out the custom attributes available and give a brief description of each attribute (one line description)

Reply:
Done